api/controllers/EAppSubmittedController.js

```diff
@@ -0,0 +1,180 @@
+const sails = require('sails');
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({ region: 'eu-west-2' });
```

Comment:
Do not hard code the region in everywhere. Look it up from an env var

Reply:
This is really helpful. Thanks for sharing this with me 👍